packages/terra-demographics-banner/src/DemographicsBanner.scss

```diff
@@ -7,7 +7,7 @@
   margin-bottom: 0.714em;
   padding: 0.714em;
 
-  @media screen and (min-width: $terra-small-breakpoint) {
+  @media screen and (min-width: 768px) {
```

Comment:
Yes, all variables that consumers could theme are being hard-coded. This is to make the switch to a css-in-js solution more passive for consumers so they don't have to be concerned with migrating a sass file with variables to a JSON file for variables.
Once we've switched over to a css-in-js solution, we'll add the theming capabilities back as a enhancement which will be passive for consumers. That work is scheduled to start in July.